psydac/linalg/stencil.py

```diff
@@ -1917,9 +1853,8 @@ def __init__( self, V, W, s_d, s_c, d_axis, c_axis, d_ext, c_ext, *, flip=None,
         self._args         = args.copy()
         self._func         = self._dot
 
-        self._transpose_args_null = self._prepare_transpose_args()
-        self._transpose_args      = self._transpose_args_null.copy()
-        self._transpose_func      = self._transpose
+        self._transpose_args= self._prepare_transpose_args()
+        self._transpose_func      = eval(f'interface_transpose_{self._ndim}d')
```

Comment:
Please align the equal signs and remove the useless empty spaces.

Reply:
[d81b27f](https://github.com/pyccel/psydac/pull/305/commits/d81b27f827b9ed72705d739d76ff1fa374119e2b)